lib/std/crypto/Certificate/Bundle.zig

```diff
@@ -90,6 +88,34 @@ pub fn rescanLinux(cb: *Bundle, gpa: Allocator) !void {
     cb.bytes.shrinkAndFree(gpa, cb.bytes.items.len);
 }
 
+pub fn rescanWindows(cb: *Bundle, gpa: Allocator) !void {
+    cb.bytes.clearRetainingCapacity();
+    cb.map.clearRetainingCapacity();
+
+    const store = os.windows.crypt32.CertOpenSystemStoreA(null, "ROOT");
```

Comment:
This function may fail. Please check for an error after this call.

Reply:
Ok. There is no documentation about possible errors, so I used `unexpectedError`. Also, used unreachable in closeStore function in defer clause (closeStore also can return error), because in any case, it will free memory (should I change it to do noop?). I hope this is ok. Also, I don't think that CertOpenSystemStoreW can return error, if i provide string other than `"ROOT"`, it doesn't seem to return any error. But in any case, i handled possible errors, i think.

Reply:
> CertOpenSystemStoreW 
> Errors from the called function [CertOpenStore](https://learn.microsoft.com/en-us/windows/desktop/api/wincrypt/nf-wincrypt-certopenstore) are propagated to this function.

Which includes `ERROR_FILE_NOT_FOUND`.

> CertOpenStore
> [CreateFile](https://learn.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-createfilea), [ReadFile](https://learn.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-readfile), or registry errors might be propagated and their error codes returned.

Alongside these all of the error codes associated with ReadFile and WriteFile.